theano/tensor/nnet/tests/test_conv3d2d.py

```diff
@@ -153,7 +152,6 @@ def test_conv3d(mode=mode_without_gpu, shared=theano.tensor._shared):
     newconv3d = theano.function([], [],
                                 updates={s_output: out},
                                 mode=mode)
-    check_diagonal_subtensor_view_traces(newconv3d)
```

Comment:
In these two cases, the ops (DiagonalSubtensor, IncDiagonalSubtensor) are not in the graph, do you confirm it is normal or should I investigate more?